lib/usd/translators/meshWriter.h

```diff
@@ -44,125 +44,37 @@ PXR_NAMESPACE_OPEN_SCOPE
-            TfToken* interpolation,
-            VtArray<int>* colorSetAssignmentIndices,
-            MFnMesh::MColorRepresentation* colorSetRep,
-            bool* clamped);
-
-    bool _createAlphaPrimVar(
-            UsdGeomGprim& primSchema,
-            const TfToken& name,
-            const UsdTimeCode& usdTime,
-            const VtArray<float>& data,
-            const TfToken& interpolation,
-            const VtArray<int>& assignmentIndices,
-            bool clamped);
-
-    bool _createRGBPrimVar(
-            UsdGeomGprim& primSchema,
-            const TfToken& name,
-            const UsdTimeCode& usdTime,
-            const VtArray<GfVec3f>& data,
-            const TfToken& interpolation,
-            const VtArray<int>& assignmentIndices,
-            bool clamped);
-
-    bool _createRGBAPrimVar(
-            UsdGeomGprim& primSchema,
-            const TfToken& name,
-            const UsdTimeCode& usdTime,
-            const VtArray<GfVec3f>& rgbData,
-            const VtArray<float>& alphaData,
-            const TfToken& interpolation,
-            const VtArray<int>& assignmentIndices,
-            bool clamped);
-
-    /// Adds displayColor and displayOpacity primvars using the given color,
-    /// alpha, and assignment data if the \p primSchema does not already have
-    /// authored opinions for them.
-    bool _addDisplayPrimvars(
-            UsdGeomGprim& primSchema,
-            const UsdTimeCode& usdTime,
-            const MFnMesh::MColorRepresentation colorRep,
-            const VtArray<GfVec3f>& RGBData,
-            const VtArray<float>& AlphaData,
-            const TfToken& interpolation,
-            const VtArray<int>& assignmentIndices,
-            const bool clamped,
-            const bool authored);
-
-    /// Sets the primvar \p primvar at time \p usdTime using the given
-    /// \p indices (can be empty) and \p values.
-    /// The \p defaultValue is used to pad the \p values array in case
-    /// \p indices contains unassigned indices (i.e. indices < 0) that need a
-    /// corresponding value in the array.
-    ///
-    /// When authoring values at a non-default time, _SetPrimvar() might
-    /// unnecessarily pad \p values with \p defaultValue in order to guarantee
-    /// that the primvar remains valid during the export process. In that case,
-    /// the expected value of UsdGeomPrimvar::ComputeFlattened() is still
-    /// correct (there is just some memory wasted).
-    /// In order to cleanup any extra values and reclaim the wasted memory, call
-    /// _CleanupPrimvars() at the end of the export process.
-    void _SetPrimvar(
-            const UsdGeomPrimvar& primvar,
-            const VtIntArray& indices,
-            const VtValue& values,
-            const VtValue& defaultValue,
-            const UsdTimeCode& usdTime);
-
-    /// Cleans up any extra data authored by _SetPrimvar().
-    void _CleanupPrimvars();
+    /// Cleans up any extra data authored by SetPrimvar().
```

Comment:
Again, this is actually `setPrimvar()` from `meshWriteUtils.cpp` that's not directly visible by this class.